examples/05_advanced/plot_ica_resting_state.py

```diff
@@ -45,10 +43,6 @@
                      mask_strategy='epi', standardize=True)
 data_masked = masker.fit_transform(func_filename)
 
-# Concatenate all the subjects
-# fmri_data = np.concatenate(data_masked, axis=1)
```

Comment:
To me it seems this is for when the users decide to try this with more than one subject as indicated in the original text.

Reply:
It wouldn't work as written. We still only take `func[0]`. We'd have to rework more of the example to allow for multiple subjects.

Reply:
Okay, I misunderstood this. Thanks.